sidecar/app/routes/start.py

```diff
@@ -47,11 +69,17 @@ def start_ipa_helper(
     multi_threading: Annotated[bool, Form()],
     disable_metrics: Annotated[bool, Form()],
     background_tasks: BackgroundTasks,
+    request: Request,
 ):
     # pylint: disable=too-many-arguments
+    query_manager = request.app.state.QUERY_MANAGER
```

Comment:
it's only twice, and it's because the report collector has a different path than the helpers. if anything, we may just want to put those in the same route.

for now, I'm not too worried about a 2x duplication, but if it we're more than that, I'd agree we should handle it elsewhere.